app/src/main/res/values/strings.xml

```diff
@@ -907,5 +908,16 @@
     <string name="mark_all_as_read">Mark all as read</string>
     <string name="global_search_contacts_groups">Contacts and Groups</string>
     <string name="global_search_messages">Messages</string>
+    <string name="activity_message_requests_title">Message Requests</string>
+    <string name="message_requests_send_notice">Sending a message to this user will automatically accept their message request and reveal your Session ID.</string>
```

Comment:
Is this resource reserved for the ID blinding?

Reply:
In the design this text is shown above the Approve/Decline buttons so should be applicable now?